### PR TITLE
Added support for Roman numeral Unicode characters

### PR DESCRIPTION
**Pull request by jkporter**

### CODE REVIEW

File: src/Humanizer/RomanNumeralExtensions.cs [hunk 1]
```diff
@@ -110,4 +188,11 @@ private static bool IsInvalidRomanNumeral(string input)
             return !ValidRomanNumeral.IsMatch(input);
         }
     }
+
+    public enum RomanNumeralCharacters
```
Comment (mexx) [inline src/Humanizer/RomanNumeralExtensions.cs]: Please add documentation comments for this enum and its members.

File: src/Humanizer/RomanNumeralExtensions.cs [hunk 2]
```diff
@@ -81,19 +155,23 @@ public static int FromRoman(this string input)
         /// Converts the input to Roman number
         /// </summary>
         /// <param name="input">Integer input</param>
+        /// <param name="characters">The characters used for Roman numerals</param>
```
Comment (mexx) [inline src/Humanizer/RomanNumeralExtensions.cs]: Please note the default value used.

**Comment by mexx** [comment]
@jkporter Thank you for the addition and PR.

I left some comments on the code.
Further could you please fix the build, running `build.cmd` shouldn't fail neither local nor on CI server.

Please document the new possiblity in `readme.md` and also add your PR to the `release_notes.md`.
See [CONTRIBUTING.md](https://github.com/MehdiK/Humanizer/blob/master/CONTRIBUTING.md)

**Comment by MehdiK** [comment]
Any updates on this @jkporter? It's been inactive for a month now.

**Comment by jkporter** [comment]
Yes, I still think the additions are worthwhile. It's fallen off my radar a bit but I do intend finish the work per the acceptance criteria.

Sent from Outlook<http://taps.io/outlookmobile>




On Thu, May 28, 2015 at 7:58 PM -0700, "Mehdi Khalili" <notifications@github.com<mailto:notifications@github.com>> wrote:


Any updates on this @jkporter<https://github.com/jkporter>? It's been inactive for a month now.

-

**Comment by jkporter** [comment]
After some research with regard to Unicode compatibility characters and Roman numerals, I've decided to close this pull request.

**Pull request closed without merging**